pkg/controller/controller.go

```diff
@@ -209,27 +216,13 @@ func (c *Controller) Run(stopCh <-chan struct{}) error {
 	// Enable API readiness endpoint, informers are started and default conf is available.
 	c.api.EnableReadiness()
 
-	for {
-		select {
-		case <-stopCh:
-			c.logger.Info("Shutting down workers")
-			return nil
-
-		case <-c.configRefreshChan:
-			// Reload the configuration.
-			topo, err := c.topologyBuilder.Build(c.ignoredResources)
-			if err != nil {
-				c.logger.Errorf("Unable to build dynamic configuration: %v", err)
-				continue
-			}
-
-			conf := c.provider.BuildConfig(topo)
-
-			if !reflect.DeepEqual(c.currentConfiguration.Get(), conf) {
-				c.currentConfiguration.Set(conf)
-			}
-		}
-	}
+	// Start to poll work from the queue.
+	go wait.Until(c.runWorker, time.Second, stopCh)
```

Comment:
It's not possible, the `runWorker` call blocks until a message is available in the queue as explained in `Get` documentation of the work queue:

>// Get blocks until it can return an item to be processed. If shutdown = true,
// the caller should end their goroutine. You must call Done with item when you
// have finished processing it.